Formula/man-db.rb

```diff
@@ -0,0 +1,45 @@
+class ManDb < Formula
+  desc "Modern, featureful implementation of the Unix man page system"
+  homepage "https://nongnu.org/man-db/"
+  url "https://download.savannah.nongnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  depends_on "libpipeline"
+
+  def install
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-systemdtmpfilesdir=no",
+                          "--with-systemdsystemunitdir=no",
+                          "--disable-cache-owner",
+                          "--disable-setuid"
+    ENV.append_to_cflags "-Wl,-flat_namespace,-undefined,suppress"
+    system "make", "CFLAGS=#{ENV.cflags}"
+    system "make", "install"
+  end
+
+  test do
+    ENV["PAGER"] = "cat"
+    ENV["MANWIDTH"] = "80"
+    expected = <<~EOS
+      TRUE(1)                   BSD General Commands Manual                  TRUE(1)
+
+      NAME
+           true -- Return true value.
+
+      SYNOPSIS
+           true
+
+      DESCRIPTION
+           The true utility always returns with exit code zero.
+
+      SEE ALSO
+           csh(1), sh(1), false(1)
+
+      STANDARDS
+           The true utility conforms to IEEE Std 1003.2-1992 (``POSIX.2'').
+
+      BSD                              June 27, 1991                             BSD
+    EOS
+    assert_match expected, shell_output("#{bin}/man true")
```

Comment:
We can probably match two substrings from this, rather than the full page (to make the test more robust).

```
    assert_match "BSD General Commands Manual", output
    assert_match "The true utility always returns with exit code zero", output
```

Reply:
Done.